### PR TITLE
add osf client details to secrets file

### PR DESCRIPTION
**Pull request by whunter**
used to inject the client id and the client secret into the application.

### CODE REVIEW

File: ansible/roles/sufia/defaults/main.yml [hunk 1]
```diff
@@ -20,5 +20,9 @@ graylog_protocol: udp
 graylog_network_locality: WAN
 graylog_facility: "{{ project_name }}"
 graylog_verbosity: "info"
+# OSF API settings
+osf_client_id: ""
```
Comment (pmather) [inline ansible/roles/sufia/defaults/main.yml]: Shouldn't this be `project_osf_client_id`?

File: ansible/roles/sufia/defaults/main.yml [hunk 2]
```diff
@@ -20,5 +20,9 @@ graylog_protocol: udp
...
+osf_client_secret: ""
```
Comment (pmather) [inline ansible/roles/sufia/defaults/main.yml]: Shouldn't this be `project_osf_client_secret`?

**Review by pmather — CHANGES_REQUESTED** [review]
The default variables for the `sufia` role are not used in the template or anywhere else.  They should be changed as suggested.

File: ansible/roles/sufia/defaults/main.yml (hunk 1 above)
Comment (whunter) [reply ansible/roles/sufia/defaults/main.yml]: @pmather Change made

File: ansible/roles/sufia/defaults/main.yml (hunk 2 above)
Comment (whunter) [reply ansible/roles/sufia/defaults/main.yml]: Change made

File: ansible/roles/sufia/templates/secrets.yml.j2 [hunk 3]
```diff
@@ -46,7 +46,11 @@ default: &default
   # secret_key_base is typically set to a long, random string, such as the output
   # of "openssl rand -hex 64" (or the output of the "rake secret" task).
   secret_key_base: abad1dea
-
+  # OSF client secrets
+  osf:
```
Comment (tingtingjh) [inline ansible/roles/sufia/templates/secrets.yml.j2:50]: osf: &osf?

**Review by tingtingjh — APPROVED** [review]

**Pull request merged**